### DISCUSSION TITLE
VDH NO LONGER WORK - EMAIL NO LONGER WORKS!!!!!

### DISCUSSION THREAD

**Discussion by 1s3k4I** [aclap-dev/video-downloadhelper#378] [License & Purchase requests]
### What's your issue?

 I BOUGHT PREMIUM, HAVEN'T BEEN ABLE TO USE APP FOR A LONG TIME, KEEP GETTING "No license has been registered"...
TOTALLY PISSED OFF! WTF! - I NO LONGER CARE IF IT'S THROUGH CHROME, FIREFOX OR EDGE, I GOT PREMIUM THROUGH CHROME WITH THE WRONG EMAIL NOT REALISING HOW STUPID THE PROCESS IS & THAT IT WOULD ONLY WORK THROUGH CHROME & NOT MY PRIMARY BROWSER FIREFOX...

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
https://github.com/aclap-dev/video-downloadhelper/wiki/Old-Chrome-licenses-(key-missing)
